chart/values.yaml

```diff
@@ -46,11 +46,14 @@ es-master: &es
     - name: elasticsearch-certificates
       secretName: "{{.Release.Name}}-elasticsearch-cert"
       path: /usr/share/elasticsearch/config/certs
-  volumeClaimTemplate:
```

Comment:
keep the `volumeClaimTemplate` template block, you will still need it for a real/production cluster. The `persistence.enabled` flag should take of using or not using `volumeClaimTemplate`.